app/src/main/res/layout/create_playlist_dialog.xml

```diff
@@ -32,22 +31,24 @@
         tools:layout_editor_absoluteY="30dp" />
 
     <Button
-        android:id="@+id/btnOk"
+        android:id="@+id/btnCreate"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_below="@+id/etPlaylistName"
+        android:layout_below="@+id/etNewPlaylistName"
         android:layout_alignParentEnd="true"
         android:layout_marginTop="49dp"
         android:layout_marginEnd="53dp"
-        android:text="Ok" />
+        android:text="Create" />
```

Comment:
move text to string resources - explanation:
https://developer.android.com/guide/topics/resources/string-resource